### ISSUE TITLE
Error on make bootstrap

### ISSUE THREAD

**Issue by Throckmortra** [mongodb/vault-plugin-secrets-mongodbatlas#27]
error:

```
make bootstrap
Installing/Updating github.com/mitchellh/gox
Installing/Updating github.com/golang/dep/cmd/dep
go: finding github.com/sdboyer/constext latest
go: finding golang.org/x/sync latest
go: finding github.com/nightlyone/lockfile latest
go: finding golang.org/x/sys latest
# github.com/golang/dep/gps
../../../../pkg/mod/github.com/golang/dep@v0.5.4/gps/constraint.go:149:4: undefined: semver.Constraint
make: *** [bootstrap] Error 2
```

version:

```
go version go1.13.6 darwin/amd64
```

**Comment by Throckmortra** [comment]
also any hint towards timeline for this being added to core plugins? :)

**Comment by themantissa** [comment]
@Throckmortra - sorry for the delay, we are moving code between repos so we expect issues for a bit.  Regarding hints expect to see info very soon on this :)

**Comment by themantissa** [comment]
@Throckmortra wanted to let you know this work is now included in the 1.4Beta1 version of Vault released by HashiCorp and will be in the GA 1.4 Vault version.  I'll close this PR as you won't need to build this on your own now.
